src/Components/Web.JS/src/Services/NavigationUtils.ts

```diff
@@ -93,35 +93,19 @@ function canProcessAnchor(anchorTarget: HTMLAnchorElement) {
 }
 
 function findAnchorTarget(event: MouseEvent): HTMLAnchorElement | null {
-  // _blazorDisableComposedPath is a temporary escape hatch in case any problems are discovered
-  // in this logic. It can be removed in a later release, and should not be considered supported API.
-  const path = !window['_blazorDisableComposedPath'] && event.composedPath && event.composedPath();
+  const path = event.composedPath && event.composedPath();
   if (path) {
     // This logic works with events that target elements within a shadow root,
     // as long as the shadow mode is 'open'. For closed shadows, we can't possibly
     // know what internal element was clicked.
     for (let i = 0; i < path.length; i++) {
       const candidate = path[i];
-      if (candidate instanceof Element && candidate.tagName === 'A') {
+      if (candidate instanceof HTMLAnchorElement || candidate instanceof SVGAElement) {
         return candidate as HTMLAnchorElement;
```

Comment:
So this can either be a `HTMLAnchorElement` or `SVGAElement` now. Should this method return `HTMLAnchorElement | SVGAElement | null` instead? Is the cast then still needed because of the `instanceof` checks before?

Reply:
I agree it is a bit dubious that the function claims to return `HTMLAnchorElement | null` if it actually returns `HTMLAnchorElement | SVGAElement | null`. However I don't know that it would make any difference in practice.

@surayya-MS Does it work to change the return type to be more accurate now, or does that force a lot of disruption to the caller?

Reply:
I added `SVGAElement ` as return type too. Works fine